### ISSUE TITLE
Failed to execute script main

### ISSUE THREAD

**Issue by vinsibal** [ElliotGarbus/Mercury7-Editor-and-Librarian#1]
When launching the app I get the following dialog showing "Failed to execute script main" then the app terminates

**Comment by ElliotGarbus** [comment]
On what OS?

**Comment by vinsibal** [comment]
This is on windows 10

**Comment by ElliotGarbus** [comment]
This suggests that something corrupted the install.  Are you using a virus scanner? Try turning off your virus scan during the install, or use a different scanner.  I have no problem with McAfee.

**Comment by ghost** [comment]
Having this same issue using Webroot. I'm writing them now to try and get these editors whitelisted.

**Comment by pfiled** [comment]
Are there any other suggestions for this? I'm having this issue with both the Mercury7 and Enzo editors on Win 10, I also disabled Windows defender real time scanning, and tried re-installing.

**Comment by ghost** [comment]
> Are there any other suggestions for this? I'm having this issue with both the Mercury7 and Enzo editors on Win 10, I also disabled Windows defender real time scanning, and tried re-installing.

Are you using antivirus software? My issues were directly related to Webroot quarantining key files for the editors to run. Once I got Webroot to add those files to a whitelist, things have gone smoothly.

**Comment by pfiled** [comment]
> Are you using antivirus software? My issues were directly related to Webroot quarantining key files for the editors to run. Once I got Webroot to add those files to a whitelist, things have gone smoothly.

Just defender, which I turned off, and I added the program folder to the exclusions list . Can you share which files were being blocked? Is there another way to execute the program to get some debug information?

**Comment by ElliotGarbus** [comment]
There should be a log file located at C:\Users\YourUserName\.kivy\logs
There might be some information on what is not loading, assuming the program is getting this far.
If you get the log file, please post it here.

**Comment by pfiled** [comment]
Here you go, I'm happy to test any builds or make any changes needed to assist troubleshooting. Thank you!

```
[INFO   ] Logger: Record log in C:\Users\danie\.kivy\logs\kivy_20-07-21_6.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "C:\Program Files (x86)\Mercury7 Editor and Librarian\kivy\__init__.pyc"
[INFO   ] Python: v3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 19:29:22) [MSC v.1916 32 bit (Intel)]
[INFO   ] Python: Interpreter at "C:\Program Files (x86)\Mercury7 Editor and Librarian\Mercury7Edit.exe"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.6.0 - Build 26.20.100.8141'>
[INFO   ] GL: OpenGL vendor <b'Intel'>
[INFO   ] GL: OpenGL renderer <b'Intel(R) Iris(R) Plus Graphics'>
[INFO   ] GL: OpenGL parsed version: 4, 6
[INFO   ] GL: Shading version <b'4.60 - Build 26.20.100.8141'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <32>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] GL: NPOT texture support is available
[WARNING] Both Window.minimum_width and Window.minimum_height must be bigger than 0 for the size restriction to take effect.
[INFO   ] Base: Start application main loop
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "main.py", line 762, in <module>
[WARNING] stderr:   File "site-packages\kivy\app.py", line 855, in run
[WARNING] stderr:   File "site-packages\kivy\base.py", line 504, in runTouchApp
[WARNING] stderr:   File "site-packages\kivy\core\window\window_sdl2.py", line 747, in mainloop
[WARNING] stderr:   File "site-packages\kivy\core\window\window_sdl2.py", line 479, in _mainloop
[WARNING] stderr:   File "site-packages\kivy\base.py", line 339, in idle
[WARNING] stderr:   File "site-packages\kivy\clock.py", line 591, in tick
[WARNING] stderr:   File "kivy\_clock.pyx", line 384, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy\_clock.pyx", line 414, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy\_clock.pyx", line 412, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy\_clock.pyx", line 167, in kivy._clock.ClockEvent.tick
[WARNING] stderr:   File "MerisCommon\meris_midi.py", line 326, in read_midi_callback
[WARNING] stderr: TypeError: __new__() missing 1 required positional argument: 'cmd'
```

**Comment by ElliotGarbus** [comment]
My suggestion is to uninstall the apps from the Windows10 Settings/Apps screen. Then:

Turn off your virus scanner
Download an editor
Install the editor
Check if it runs
Turn on your Virus scanner

Please let me know the outcome.

**Comment by mayasfinest** [comment]
Hello,
i really like your Meris Editors a lot, thanks for giving them to us :-)
I used to use the Mercury7,Hedra,Polymoon and Enzo on my Desktop PC (Win10Pro) which worked like a charm. 
Now i bought me a Windows10Pro tablet and suddenly ran into the same issues as the user above and also tried everything i could think of.
Do you know of any solution for this "failed to execute script main" issue? It seems to be quite common with other python programmed softwares...
Any help appreciated,
thanks again and stay healthy,
greetings from Germany, Jörg

**Comment by ElliotGarbus** [comment]
Jorg,
Turn off the virus scanner and reinstall the app.  Certain Virus scanners seem to block elements of the program that cause this problem.

If that does not address the issue please attach the log file.  That might provide some indication of the problem.   There should be a log file located at C:\Users\YourUserName\.kivy\logs

**Comment by mayasfinest** [comment]
Thanks Elliot for the fast reply,
as i stated, i tried all of the above. It's only Windows Defender working on my Tablet, same configuration as on my Desktop DAW. I turned it off completely, tried de- and reinstalling etc. 
This is the log from the latest start:

[INFO   ] Logger: Record log in C:\Users\Lenovo Tab\.kivy\logs\kivy_20-10-25_4.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "C:\Program Files\Mercury7 Editor and Librarian\kivy\__init__.pyc"
[INFO   ] Python: v3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 19:29:22) [MSC v.1916 32 bit (Intel)]
[INFO   ] Python: Interpreter at "C:\Program Files\Mercury7 Editor and Librarian\Mercury7Edit.exe"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.0.0 - Build 10.18.10.4491'>
[INFO   ] GL: OpenGL vendor <b'Intel'>
[INFO   ] GL: OpenGL renderer <b'Intel(R) HD Graphics'>
[INFO   ] GL: OpenGL parsed version: 4, 0
[INFO   ] GL: Shading version <b'4.00 - Build 10.18.10.4491'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "main.py", line 21, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "MerisCommon\meris_midi.py", line 2, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\mido\backends\rtmidi.py", line 8, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\rtmidi\__init__.py", line 4, in <module>
[WARNING] stderr:     
[WARNING] stderr: ImportError: DLL load failed: Das angegebene Modul wurde nicht gefunden.


**I also tried starting the Editors in Windows Safe-Mode**


This is the log from that tryout:

[INFO   ] Logger: Record log in C:\Users\Lenovo Tab\.kivy\logs\kivy_20-10-25_5.txt
[WARNING] [Config      ] Upgrading configuration in progress.
[WARNING] [Config      ] Older configuration version detected (21 instead of 20)
[INFO   ] Kivy: v1.10.1
[INFO   ] Python: v3.7.1 (v3.7.1:260ec2c36a, Oct 20 2018, 14:05:16) [MSC v.1915 32 bit (Intel)]
[INFO   ] Factory: 194 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: No framebuffers extension is supported
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'1.1.0'>
[INFO   ] GL: OpenGL vendor <b'Microsoft Corporation'>
[INFO   ] GL: OpenGL renderer <b'GDI Generic'>
[INFO   ] GL: OpenGL parsed version: 1, 1
[CRITICAL] GL: Minimum required OpenGL version (2.0) NOT found!

OpenGL version detected: 1.1

Version: b'1.1.0'
Vendor: b'Microsoft Corporation'
Renderer: b'GDI Generic'

Try upgrading your graphics drivers and/or your graphics hardware in case of problems.

The application will leave now.

**Comment by ElliotGarbus** [comment]
In windows safe mode, the proper version of OpenGL is not installed.  We can see it is installed in the normal mode.  There is nothing to learn from the safe mode install.

In the 'normal mode' there is a failure to load a module, this suggests something interfered with the installation, and a file is not where it is supposed to be.  All I can suggest is to:
uninstall
re-boot
Turn off virus scan
Install the Editor

Let me know if that fixes things.  If that doesn't work there may be a few other things to try - but I'm not optimistic they will fix the issues.

**Comment by mayasfinest** [comment]
Hi,
still same error message...
I also tried installing in safe mode, starting as admin, starting with different compatibility changes, rebooting before re-installing, and a different install path...
Is there a possibility to put the missing things manually to their expected position on the HD?

**Comment by vinsibal** [comment]
Curious, what's the biggest difference between the Polymoon editor and this one?  I had no issues installing and launching the Polymoon editor but haven't been able to get this Mercury Editor to launch.

**Comment by mayasfinest** [comment]
> Curious, what's the biggest difference between the Polymoon editor and this one? I had no issues installing and launching the Polymoon editor but haven't been able to get this Mercury Editor to launch.

I have the Polymoon, Enzo, Hedra and Mercury7 Editors running with my big DAW machine with no problems and now wanted to place them on a dedicated tablet controller. Sadly all have the same problem :-(

**Comment by ElliotGarbus** [comment]
@vinsibal As you might expect the guts of the editors are the same.  The differences are largely the presentation of the editor panel, each one controls the parameters in custom way.  Please share your log file.  It will be interesting to see if we have the same issue as @mayasfinest or if something different is going on.

@mayasfinest I use a library that does the low level midi communications... the problem is happening in a library that it uses.  If your willing to put up with a few drops I can try the following... I'll check for updates of library that is having the problem, and do a new build for you to try.

**Comment by ElliotGarbus** [comment]
@pfiled Did you ever get this working?  It looks like the editor is receiving invalid data from the M7, based on your posted log.  A HW reset of the M7 should fix this.

**Comment by mayasfinest** [comment]
These are the 4 logs of the 4 editors i have installed, reproducing the same kind of error on my machine. 
If i could provide you any details about my tablet, just tell me...
I would love to have your editors running, thanks for your effort !

[INFO   ] Logger: Record log in C:\Users\Lenovo Tab\.kivy\logs\kivy_20-10-25_31.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "C:\Program Files\Polymoon Editor and Librarian\kivy\__init__.pyc"
[INFO   ] Python: v3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 21:26:53) [MSC v.1916 32 bit (Intel)]
[INFO   ] Python: Interpreter at "C:\Program Files\Polymoon Editor and Librarian\PolymoonEdit.exe"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.0.0 - Build 10.18.10.4491'>
[INFO   ] GL: OpenGL vendor <b'Intel'>
[INFO   ] GL: OpenGL renderer <b'Intel(R) HD Graphics'>
[INFO   ] GL: OpenGL parsed version: 4, 0
[INFO   ] GL: Shading version <b'4.00 - Build 10.18.10.4491'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "main.py", line 21, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "MerisCommon\meris_midi.py", line 2, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\mido\backends\rtmidi.py", line 8, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\rtmidi\__init__.py", line 4, in <module>
[WARNING] stderr:     
[WARNING] stderr: ImportError: DLL load failed: Das angegebene Modul wurde nicht gefunden.



[INFO   ] Logger: Record log in C:\Users\Lenovo Tab\.kivy\logs\kivy_20-10-25_30.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "C:\Program Files\Enzo Editor and Librarian\kivy\__init__.pyc"
[INFO   ] Python: v3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 19:29:22) [MSC v.1916 32 bit (Intel)]
[INFO   ] Python: Interpreter at "C:\Program Files\Enzo Editor and Librarian\EnzoEdit.exe"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.0.0 - Build 10.18.10.4491'>
[INFO   ] GL: OpenGL vendor <b'Intel'>
[INFO   ] GL: OpenGL renderer <b'Intel(R) HD Graphics'>
[INFO   ] GL: OpenGL parsed version: 4, 0
[INFO   ] GL: Shading version <b'4.00 - Build 10.18.10.4491'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "main.py", line 21, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "MerisCommon\meris_midi.py", line 2, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\mido\backends\rtmidi.py", line 8, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\rtmidi\__init__.py", line 4, in <module>
[WARNING] stderr:     
[WARNING] stderr: ImportError: DLL load failed: Das angegebene Modul wurde nicht gefunden.


[INFO   ] Logger: Record log in C:\Users\Lenovo Tab\.kivy\logs\kivy_20-10-25_29.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "C:\Program Files\Mercury7\kivy\__init__.pyc"
[INFO   ] Python: v3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 19:29:22) [MSC v.1916 32 bit (Intel)]
[INFO   ] Python: Interpreter at "C:\Program Files\Mercury7\Mercury7Edit.exe"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.0.0 - Build 10.18.10.4491'>
[INFO   ] GL: OpenGL vendor <b'Intel'>
[INFO   ] GL: OpenGL renderer <b'Intel(R) HD Graphics'>
[INFO   ] GL: OpenGL parsed version: 4, 0
[INFO   ] GL: Shading version <b'4.00 - Build 10.18.10.4491'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "main.py", line 21, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "MerisCommon\meris_midi.py", line 2, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\mido\backends\rtmidi.py", line 8, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\rtmidi\__init__.py", line 4, in <module>
[WARNING] stderr:     
[WARNING] stderr: ImportError: DLL load failed: Das angegebene Modul wurde nicht gefunden.


[INFO   ] Logger: Record log in C:\Users\Lenovo Tab\.kivy\logs\kivy_20-10-25_28.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "C:\Program Files\Hedra Editor and Librarian\kivy\__init__.pyc"
[INFO   ] Python: v3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 19:29:22) [MSC v.1916 32 bit (Intel)]
[INFO   ] Python: Interpreter at "C:\Program Files\Hedra Editor and Librarian\HedraEdit.exe"
[INFO   ] Logger: Purge log fired. Analysing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.0.0 - Build 10.18.10.4491'>
[INFO   ] GL: OpenGL vendor <b'Intel'>
[INFO   ] GL: OpenGL renderer <b'Intel(R) HD Graphics'>
[INFO   ] GL: OpenGL parsed version: 4, 0
[INFO   ] GL: Shading version <b'4.00 - Build 10.18.10.4491'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "main.py", line 21, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "MerisCommon\meris_midi.py", line 2, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\mido\backends\rtmidi.py", line 8, in <module>
[WARNING] stderr:   File "c:\users\ellio\kivy_venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
[WARNING] stderr:   File "site-packages\rtmidi\__init__.py", line 4, in <module>
[WARNING] stderr:     
[WARNING] stderr: ImportError: DLL load failed: Das angegebene Modul wurde nicht gefunden.

**Comment by mayasfinest** [comment]
PS: I also tried installing it in the root dir of an USB stick, makes no difference...

**Comment by ElliotGarbus** [comment]
@mayasfinest I'll put together a few builds and see if we can fix this issue.  It is a strange one.
Just as an FYI - it looks like your graphics driver is behind a few revs, you might want to update it.  I don't think it will fix this issue.

**Comment by mayasfinest** [comment]
> @mayasfinest I'll put together a few builds and see if we can fix this issue. It is a strange one.
> Just as an FYI - it looks like your graphics driver is behind a few revs, you might want to update it. I don't think it will fix this issue.

Thanks a lot, i appreciate it very much.
The tablet (i just bought blank) just filled itself with windows standard drivers, i will now scan for dedicated Lenovo drivers and let you know if it makes any difference...

**Comment by ElliotGarbus** [comment]
you might want to try: https://www.intel.com/content/www/us/en/support/intel-driver-support-assistant.html

**Comment by mayasfinest** [comment]
Eureka!
It may have been some outdated special driver Windows-update did not find (not the intelgraphic one, there was no newer) - now everything works as expected. With the old installs and with the Windows Defender fully activated.
Thanks for your efforts and stay healthy!

**Comment by ElliotGarbus** [comment]
Awsome!  Thanks for letting me know.  I'm glad you enjoy the editors!

-Stay Healthy!

**Comment by vinsibal** [comment]
I finally figured out the issue after realizing where the log files are from a previous post here:  C:\Users\YourUserName.kivy\logs.

[WARNING] Both Window.minimum_width and Window.minimum_height must be bigger than 0 for the size restriction to take effect.
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "main.py", line 762, in <module>
[WARNING] stderr:   File "site-packages\kivy\app.py", line 829, in run
[WARNING] stderr:   File "main.py", line 673, in build
[WARNING] stderr:   File "MerisCommon\thelibrary.py", line 119, in read_database
[WARNING] stderr:   File "MerisCommon\thelibrary.py", line 134, in rename_lib
[WARNING] stderr:   File "MerisCommon\thelibrary.py", line 95, in write_database
[WARNING] stderr: FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\vincent\\Documents\\Mercury7 Edit Presets and Library/library.json'

When I did a windows upgrade a while back it created a folder "vincent.000" and transferred all the files over.  However %username% was still "vincent".  I did a quick fix by just copying over the library.json file to the folder above and it worked from there.
